index.html

```diff
@@ -9137,6 +9138,20 @@ <h2 id="AudioWorkletNodeOptions">
                   <a data-link-for="AudioNode">numberOfOutputs</a> attribute.
                 </dd>
                 <dt>
+                  <code><dfn>outputChannelCount</dfn></code> of type
+                  <span class="idlAttrType"><code>sequence&lt;unsigned
+                  long&gt;</code></span>
+                </dt>
+                <dd>
+                  This is used to configure the number of channels in each
+                  output. <code>IndexSizeError</code> MUST be thrown if the
```

Comment:
I think you still need to say a little bit more on what this means.  Something like "outputChannelCount[n]" specifies the number of channels for output number n". Or something.